dev-util/clinfo/metadata.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+    <maintainer type="person">
+        <email>tomboy64@sina.cn</email>
+        <name>M.B.</name>
+        <description>Maintainer, assign bugs to him</description>
```

Comment:
The description is unnecessary. Bugs are always assigned to the first person on the list.